fs/procfs/fs_procfsproc.c

```diff
@@ -2032,7 +2032,7 @@ static int proc_stat(const char *relpath, struct stat *buf)
 
   if (strncmp(relpath, "self", 4) == 0)
     {
-      tmp = (unsigned long)getpid();    /* Get the PID of the calling task */
+      tmp = (unsigned long)gettid();    /* Get the PID of the calling task */
```

Comment:
```suggestion
      tmp = gettid();    /* Get the TID of the calling task */
```

Reply:
it seem the variable tmp is defined as unsigned long
So I think we should keep this to avoid warning